mesop/components/select/select.py

```diff
@@ -35,23 +35,33 @@ class SelectOpenedChangeEvent(MesopEvent):
 
 @dataclass(kw_only=True)
 class SelectSelectionChangeEvent(MesopEvent):
-  """Event representing a change in the select component's value.
+  """Event representing a change in the select component's value(s).
 
   Attributes:
```

Comment:
Just double checking: are the pydocs generated correctly for `value`?

Reply:
Good call. I didn't consider how the property would be rendered. Yeah looks like it.

<img width="748" alt="Screenshot 2024-06-19 at 5 10 59 PM" src="https://github.com/google/mesop/assets/539889/e250ac2d-2e8e-4a99-88f4-b48ddb969593">